Code_of_Conduct.md

```diff
@@ -42,4 +42,89 @@ If you think the tangent is an important and pertinent issue, please bring it up
 
 By making an effort to follow these guidelines, we will encourage more contribution to our projects, and our discussions will be friendlier and reach conclusions more easily.
 
+## Instruments of Good Works
```

Comment:
Can you add something signifying this is an addition to the CoC and not original?

Reply:
I concur

Reply:
I have made a change with the most recent commit.